lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require 'menu'
+
+class Order
+
+  include Menu
+
+  attr_reader :item, :order
+
+  def initialize
+    @order = []
+  end
+
+  def add_item(item)
+    @item = item
+    if item_in_order? then
+      add_quantity
+      return items
+    else
+      add_new_item
+      return items
+    end
+  end
+
+  def items
+    @order.map { |item| "#{item[:item]} #{item[:quantity]} * #{item[:price]} #{sprintf "%.2f", (item[:quantity]*item[:price].to_f)}" }
+  end
+
+  private
+
+  def menu_item
+    index = MENU.index {|h| h[:item] == item }
+    MENU[index]
+  end
```

Comment:
This method reaches into the `MENU` constant. Could you extract this method into the `Menu` module?